### ISSUE TITLE
Split client ZIP

### ISSUE THREAD

**Issue by Jeeppler** [mercedes-benz/sechub#394]
Currently, we offer one single `sechub-cli-x.x.x.zip` with all clients and the documentation. However, most users only need a single client for a specific platform and operating system. 

The problem is, that each user has to download the entire ZIP file, extract it and install the needed client. It would be easier for the user, if each client can be downloaded separately. For example `sechub-cli-windows-x.x.x.zip` or `sechub-cli-macos.x.x.x.zip`. To make it easy for the user to decide which version to pick, the ZIP files should be named after the operating system. The most common one should be named just after the platform, whereas the more specific one should have additional information in the name. For example:

- `sechub-cli-windows-x.x.x.zip` ->  Windows 64bit client, assuming that today almost everybody uses a Windows 64 bit version
- `sechub-cli-windows-32bit.x.x.x.zip` -> Windows 32bit client, which should only be necessary for older operating systems.

Furthermore, the client documentation should not be included as PDF. As the PDF is not necessary for users embedding the CLI client into a CI/CD pipeline. The PDF should be offered separately. However, all clients should have a `README` file with a link to the PDF file, so that the user can download the PDF, if needed.

Splitting the `ZIP` file could be done after #385. It would be helpful to implement #373 as well, to further reduce the tools needed while installing the client.

---
<sup>Jeremias Eppler <jeremias.eppler@daimler.com>, Daimler TSS GmbH, [imprint](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md)</sup>

**Comment by Jeeppler** [comment]
Maybe using `tar.gz` for Linux would be preferred rather than using `ZIP`.

**Comment by sven-dmlr** [comment]
@Jeeppler 
_Currently, we offer one single sechub-cli-x.x.x.zip with all clients and the documentation._
The documentation is already separate.

- The current download size (all platforms in one zip) is only 17MB. This is not much.
- splitting up into separate files would break existing download scripts

For these reasons, I suggest to close this issue. (if there is a need in future for this, a new issue can be opened)
